src/Auth0.ManagementApi/Models/OrganizationCreateInvitationRequest.cs

```diff
@@ -57,7 +57,7 @@ public class OrganizationCreateInvitationRequest
         /// Whether the user will receive an invitation email (true) or no email (false), true by default
         /// </summary>
         [JsonProperty("send_invitation_email")]
-        public bool SendInvitationEmail { get; set; }
+        public bool SendInvitationEmail { get; set; } = true;
```

Comment:
As mentioned in slack, we should not provide any default value. Instead we should rely on the default value on the API.

```suggestion
        public bool? SendInvitationEmail { get; set; }
```